src/main/java/games/strategy/engine/framework/GameRunner.java

```diff
@@ -477,13 +468,9 @@ public static Image getGameIcon(final Window frame) {
     return img;
   }
 
-  public static void startNewTripleA(final Long maxMemory) {
-    startGame(System.getProperty(GameRunner.TRIPLEA_GAME_PROPERTY), null, maxMemory);
-  }
-
-  static void startGame(final String savegamePath, final String classpath, final Long maxMemory) {
+  static void startGame(final String savegamePath, final String classpath, final Optional<String> maxMemory) {
```

Comment:
Possible simplification here...  `startGame()` is only called in one place (`GameDataManager#loadGame()`).  At that call site, `maxMemory` is specified as `Optional.empty()`.  Therefore, it looks like you can remove the third argument here and remove a branch:

```diff
diff --git a/src/main/java/games/strategy/engine/framework/GameDataManager.java b/src/main/java/games/strategy/engine/framework/GameDataManager.java
index 17f1c50..0562f25 100644
--- a/src/main/java/games/strategy/engine/framework/GameDataManager.java
+++ b/src/main/java/games/strategy/engine/framework/GameDataManager.java
@@ -11,7 +11,6 @@ import java.io.ObjectOutputStream;
 import java.io.OutputStream;
 import java.io.Serializable;
 import java.util.Iterator;
-import java.util.Optional;
 import java.util.zip.GZIPInputStream;
 import java.util.zip.GZIPOutputStream;
 
@@ -108,7 +107,7 @@ public class GameDataManager {
             return null;
           }
           final boolean closeCurrentInstance = buttonPressed.equals(yesClose);
-          GameRunner.startGame(savegamePath, newClassPath, Optional.empty());
+          GameRunner.startGame(savegamePath, newClassPath);
           if (closeCurrentInstance) {
             ThreadUtil.sleep(1000);
             System.exit(0);
diff --git a/src/main/java/games/strategy/engine/framework/GameRunner.java b/src/main/java/games/strategy/engine/framework/GameRunner.java
index 766ca85..3629b78 100644
--- a/src/main/java/games/strategy/engine/framework/GameRunner.java
+++ b/src/main/java/games/strategy/engine/framework/GameRunner.java
@@ -468,13 +468,9 @@ public class GameRunner {
     return img;
   }
 
-  static void startGame(final String savegamePath, final String classpath, final Optional<String> maxMemory) {
+  static void startGame(final String savegamePath, final String classpath) {
     final List<String> commands = new ArrayList<>();
-    if (maxMemory.isPresent()) {
-      ProcessRunnerUtil.populateBasicJavaArgs(commands, classpath, maxMemory);
-    } else {
-      ProcessRunnerUtil.populateBasicJavaArgs(commands, classpath);
-    }
+    ProcessRunnerUtil.populateBasicJavaArgs(commands, classpath);
     if (savegamePath != null && savegamePath.length() > 0) {
       commands.add("-D" + GameRunner.TRIPLEA_GAME_PROPERTY + "=" + savegamePath);
     }
```